dev/src/main/java/org/eclipse/codewind/intellij/ui/CodewindToolWindow.java

```diff
@@ -195,6 +184,26 @@ public void expandLocalTree() {
         }
     }
 
+    public void expandToProject(Project project) {
+        String projectName = project.getName();
+        TreeModel model = tree.getModel();
+        Object root = model.getRoot();
+        if (model.getChildCount(root) > 0) {
+            Object connection = model.getChild(root, 0);
+            int projectCount = model.getChildCount(connection);
+            for (int i = 0; i < projectCount; i++ ) {
+                Object child = model.getChild(connection, i);
+                if (child instanceof CodewindApplication) {
+                    CodewindApplication app = (CodewindApplication)child;
+                    String name = app.getName();
+                    if (projectName.equals(name)) {
+                        tree.setSelectionPath(new TreePath(new Object[] {root, connection, app}));
```

Comment:
Should we return from the function after calling `tree.setSelectionPath()`?  Since we've found the project to select, there's no point in continuing to iterate through the tree items.

Reply:
Definitely.